BGCF_Testing.md

```diff
@@ -0,0 +1,58 @@
+The live tests allow you to test BGCF routing function, although this
```

Comment:
Can you include OFF_NET_TEL and the new skip_unless.. to the README, and link to this doc for the explanation of the OFF_NET_TEL option ?